admin/src/main/resources/io/buoyant/admin/js/spec/RouterServicesSpec.js

```diff
@@ -0,0 +1,93 @@
+"use strict";
+
+define([
+  'jQuery',
+  'lodash',
+  'src/router_service_dashboard',
+  'src/metrics_collector',
+  'spec/fixtures/metrics'
+], function($, _, RouterServices, MetricsCollector, metricsJson) {
+  describe("RouterServices", function() {
+    var $container;
+
+    var expiredMetricsJson = {
+      rt: {
+        multiplier: {
+          client: {
+            "$/inet/127.1/9092": {},
+            "$/inet/127.1/9030": {},
+            "$/inet/127.1/9029": {}
+          }
+        }
+      }
+    }
+
+    beforeEach(function () {
+      $container = $("<div />");
+    });
+
+    afterEach(function () {
+      $container.remove();
+      $container = null;
+    });
+
+    it("initializes and renders the services", function() {
+      var initialData = {
+        adder: { services: ["svc"] },
+        multiplier: { services: ["svc"]}
+      }
+      var realCollector = MetricsCollector(metricsJson);
+
+      RouterServices(realCollector, initialData, $container);
+      realCollector.__update__(metricsJson);
+
+      var $multiplierRouter = $($container.find(".router-multiplier")[0]);
+      var multiplierSvcs = $multiplierRouter.find(".svc-container");
```

Comment:
this... is a jquery object too right? might as well put a $ before it like the others. (applies to other tests too)